src/components/Profile/ProfileTile/index.js

```diff
@@ -54,6 +54,15 @@ export const ProfileTile = ({ children }) => {
 };
 
 const TileLoggedInThisMunicipality = ({ userId, userData, municipality }) => {
+
+  const getReferredUserMessage = () => {
```

Comment:
Ich fände es besser, wenn die Funktion ausgelagert wird, und nicht in beiden Komponenten dupliziert wird. dry :)

Reply:
Klar, macht vollkommen Sinn!
Ist gefixt.